cmd/poller/collector/collector.go

```diff
@@ -281,6 +281,13 @@ func (me *AbstractCollector) Start(wg *sync.WaitGroup) {
 				continue
 			}
 
+			if me.Schedule.IsStandBy() && !me.Schedule.IsTaskStandBy(task) {
```

Comment:
minor: `Msg` instead of `Msgf` since no substitution

Reply:
done